### PR TITLE
ACS-1212 Use Java from rpm

### PR DESCRIPTION
**Pull request by mteodori**

### CODE REVIEW

**Review by TheOutsSeeker — APPROVED** [review]
LGTM

**Comment by mteodori** [comment]
locking until a RPM package is available for 11.0.10 on CentOS 8 as well

File: Dockerfile [hunk 1]
```diff
@@ -18,14 +20,18 @@ ENV LANG en_US.UTF-8
...
-ADD $JAVA_PKG /usr/java/
-
-RUN export JAVA_DIR=$(ls -1 -d /usr/java/*) && \
-    ln -s $JAVA_DIR /usr/java/latest && \
-    ln -s $JAVA_DIR /usr/java/default && \
-    alternatives --install /usr/bin/java java $JAVA_DIR/bin/java 20000 && \
-    alternatives --install /usr/bin/javac javac $JAVA_DIR/bin/javac 20000 && \
-    alternatives --install /usr/bin/jar jar $JAVA_DIR/bin/jar 20000 && \
-    alternatives --install /usr/bin/keytool keytool $JAVA_DIR/bin/keytool 20000
+RUN set -eux; \
```
Comment (gavincornwell) [inline Dockerfile:23]: Have we lost the keytool utility by doing this?

**Review by gavincornwell — COMMENTED** [review]
The changes look OK in principle but I'd be nervous about merging this a few days before a major release I think it would be best to give ourselves a chance to fix any fallout.

File: Dockerfile (hunk 1 above)
Comment (mteodori) [reply Dockerfile:23]: no, it's part of the jdk rpm

**Pull request merged**